src/java.base/share/classes/sun/security/jca/Providers.java

```diff
@@ -101,7 +101,7 @@ public static Provider getSunProvider() {
     /**
      * Start JAR verification. This sets a special provider list for
      * the current thread. You MUST save the return value from this
-     * method and you MUST call stopJarVerification() with that object
+     * method, and you MUST call stopJarVerification() with that object
```

Comment:
Hmm, I'm not sure this is more grammatically correct.

Reply:
same here about dependent clause case.

Reply:
You are correct. Removing the comma.

Reply:
Maybe not. Each clause stands on its own as a complete sentence.